README.md

```diff
@@ -170,7 +170,7 @@ You can find the detailed list of all available endpoints in the [API specificat
 
 By default, Iroha provides logs in a human-readable format and prints them out to `stdout`.
 
-The logging level can be changed either via a [configuration option](./docs/source/references/config.md#loggermax_log_level) or at run-time using the `configuration` endpoint.
+The logging level can be changed either via a [configuration option](./docs/source/references/config.md#loggermax_level) or at run-time using the `configuration` endpoint.
```

Comment:
@outfardis is this a bug?

Reply:
no, why? what do you mean? from what I can see the link is updated to match changes in config.md

Reply:
I mean is that it's lumped together into `loggermax_level`

Reply:
yep, it works